### PR TITLE
Queue status page halfway done

### PR DESCRIPTION
**Pull request by maaverik**
Done with the queue details transition. Icons for check status and leave and the remaining joiner and queue details to be added soon.

Screenshot from 2020-08-02 15-53-58
Screenshot from 2020-08-02 15-53-42

### CODE REVIEW

**Comment by maaverik** [comment]
End result to be https://xd.adobe.com/view/5b8cc4d6-174a-48f4-53fa-5088ed723862-2dbd/screen/368ed3ce-47dc-4ef9-9827-54cf3f359d7c/

File: simplq/src/components/pages/Status/StatusContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    dispatch(setJoinerStep(3));
+    status = <p>Your turn is up </p>;
+  } else if (props.aheadCount === 0) {
+    status = <p>There is no one ahead of you. Please wait to be notified by the queue manager.</p>;
+  } else {
+    /* eslint-disable react/jsx-one-expression-per-line */
+    status = (
+      <p>
+        People ahead of you :<span className={styles.count}>{props.aheadCount}</span>
+      </p>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Status/StatusContainer.jsx:25]: Lets instead move the span to the next line

File: simplq/src/components/pages/Status/StatusContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (daltonfury42) [inline simplq/src/components/pages/Status/StatusContainer.jsx:1]: It's a very good split

File: simplq/src/components/common/InputField.jsx [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
 import React from 'react';
 import TextField from '@material-ui/core/TextField';
 import { StylesProvider } from '@material-ui/core/styles';
 
 // StylesProvider added to enable CSS use through className prop
-export const InputField = (props) => (
+export default (props) => (
   <StylesProvider injectFirst>
     <TextField
       placeholder={props.placeholder}
```
[the remark this answers is not in the document]
Comment (maaverik) [reply simplq/src/components/common/InputField.jsx:9]: Yeah, this looks better

File: simplq/src/components/common/utilFns.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-/* eslint-disable import/prefer-default-export */
 export const handleEnterPress = (event, keyPresshandler) => {
   if (event.key === 'Enter') {
     keyPresshandler();
   }
 };
+
+export const isQNameValid = (qname) => qname.match('^[A-Za-z0-9-]*$'); // Only letters, numbers and - allowed
```
[the remark this answers is not in the document]
Comment (maaverik) [reply simplq/src/components/common/utilFns.js:7]: Sure

File: simplq/src/components/pages/Status/StatusContainer.jsx (hunk 1 above)
Comment (maaverik) [reply simplq/src/components/pages/Status/StatusContainer.jsx:25]: Prettier makes it a single line and eslint throws an error saying it should be in different lines. I'll try configuring prettier to handle this.

File: simplq/src/components/common/InputField.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (maaverik) [reply simplq/src/components/common/InputField.jsx]: Eslint does discourage this, but I think it's fine.
`error  Prop spreading is forbidden  react/jsx-props-no-spreading`

[the remark this answers is not in the document]
Comment (daltonfury42) [reply simplq/src/components/common/InputField.jsx]: ha.. let's not do this then. I take it back

File: simplq/src/components/pages/Status/StatusContainer.jsx (hunk 1 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Status/StatusContainer.jsx:25]: ah ok. I don't know if prettier will leave it alone if we do something like this?
```
 <p>
        <span>People ahead of you :</span>
        <span className={styles.count}>{props.aheadCount}</span>
 </p>
```

File: simplq/src/components/common/InputField.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (daltonfury42) [reply simplq/src/components/common/InputField.jsx]: hmm. Upto you.

**Pull request merged**